executor/src/witgen/data_structures/identity.rs

```diff
@@ -15,28 +15,30 @@ use powdr_number::FieldElement;
 
 #[derive(Clone, Eq, PartialEq, Hash, Debug)]
 pub struct BusSend<T> {
-    /// The ID is globally unique among identities.
-    pub id: u64,
-    /// The ID of the bus interaction, used to find a matching receive.
-    pub interaction_id: AlgebraicExpression<T>,
-    /// The tuple sent to the bus, with a selector.
-    pub selected_tuple: SelectedExpressions<T>,
+    /// The identity ID is globally unique among identities.
```

Comment:
Maybe add explicitly: Matching Send and Receive operations on the same bus have different identity ids.
(because I again forgot if it is really the case :D)